src/parser/jobs/drk/modules/ActionTimeline.tsx

```diff
@@ -9,17 +9,20 @@ export class ActionTimeline extends CoreActionTimeline {
 		'DELIRIUM',
 		// oGCD Damage
 		'LIVING_SHADOW',
+		'SHADOWBRINGER',
 		'EDGE_OF_SHADOW',
 		'FLOOD_OF_SHADOW',
 		'CARVE_AND_SPIT',
 		'ABYSSAL_DRAIN',
 		'PLUNGE',
 		'SALTED_EARTH',
+		'SALT_AND_DARKNESS',
```

Comment:
Could probably group these but whatevs. Consider C&S and Abyssal too since they're on a shared CD anyway.

Reply:
I think I prefer having S&D separated since we're making separate recommendations about using that specific skill.  Similarly for keeping AD separated - it's more clear that there's an incorrect skill used if that timeline row shows up on a fight that doesn't have relevant AoE opportunities.